README.md

```diff
@@ -16,6 +16,55 @@ Official pure Rust typed client for ClickHouse DB.
 [actions-badge]: https://github.com/ClickHouse/clickhouse-rs/actions/workflows/ci.yml/badge.svg
 [actions-url]: https://github.com/ClickHouse/clickhouse-rs/actions/workflows/ci.yml
 
+## Version Compatibility
+
+### Supported Rust Versions
+
+The current MSRV (minimum supported Rust version) is **1.79**. The MSRV history for past minor releases can be found below:
+
+ * 0.13.3 - Rust 1.79 (current)
```

Comment:
It's 1.73.0 for 0.13.3

See https://github.com/ClickHouse/clickhouse-rs/blob/v0.13.3/Cargo.toml#L14

We will use 1.79.0 for the upcoming release only.